nexus_keycode/test/protocols/test_passthrough_uart.py

```diff
@@ -0,0 +1,15 @@
+from unittest import TestCase
+
+import nexus_keycode.protocols.passthrough_uart as uart
+
+
+class TestPassthroughUart(TestCase):
+    def test_passthrough_uart__expected_value_returned(self):
+        """ Test standard input into passthrough_uart and check
+            output"""
+        input_bytes = "\x00\x01\x02\x03\x04\x05\x06\x07\x08\x09\x0a\x0b\x0c\x0d\x0e\x0f"
```

Comment:
For consistency with existing test cases, I'd suggest using the following keys:

```
// 'input_bytes' is actually the 'base key' we are deriving the uart security key from
input_bytes = "\x01" * 14 + "\x43\x51"
```

The expected result is then:

```
uart_security_key = "\x12\xe4\x87\x62\x5c\x6b\x88\xf4\x1e\xe4\x0b\x16\xb4\xc9\x84\xf2"
```


from https://github.com/angaza/payg-embedded/blob/master/projects/Angaza_Portable_Library/src/angaza_test/test_payg_uart_pro.c#L49